cmd/controller-manager/app/controllermanager.go

```diff
@@ -158,6 +158,7 @@ func Run(ctx context.Context, opts *options.Options) error {
 
 	crtlmetrics.Registry.MustRegister(metrics.ClusterCollectors()...)
 	crtlmetrics.Registry.MustRegister(metrics.ResourceCollectors()...)
+	crtlmetrics.Registry.MustRegister(metrics.PoolCollectors()...)
```

Comment:
As the number of collectors increases, I think a general `initCollectors` method can be added.

Reply:
Greate idea. I think we'd better do it in another PR.